<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>796</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CourseEnroll-Schedule</string>
  </property>
  <widget class="QPushButton" name="btn_rightAll">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>80</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_left">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_right">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>180</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_leftAll">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>230</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;&lt;</string>
   </property>
  </widget>
  <widget class="QComboBox" name="cbb_course">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string>Course</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>60</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Unavailable students</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>60</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Available students</string>
   </property>
  </widget>
  <widget class="QTableWidget" name="tableUnavai">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>340</width>
     <height>441</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>170</number>
   </attribute>
   <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
    <bool>false</bool>
   </attribute>
   <attribute name="horizontalHeaderStretchLastSection">
    <bool>false</bool>
   </attribute>
   <column>
    <property name="text">
     <string>Student ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Student Name</string>
    </property>
   </column>
  </widget>
  <widget class="QTableWidget" name="tableAvai">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>80</y>
     <width>340</width>
     <height>441</height>
    </rect>
   </property>
   <attribute name="horizontalHeaderDefaultSectionSize">
    <number>113</number>
   </attribute>
   <column>
    <property name="text">
     <string>Student ID</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Student Name</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Date Enroll</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="input_room">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>550</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="placeholderText">
    <string>Room</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_lesson">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>590</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>Lesson Date</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_week">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>630</y>
     <width>321</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>30</height>
    </size>
   </property>
   <property name="placeholderText">
    <string>Week Date</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_check">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>670</y>
     <width>151</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>CHECK</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>670</y>
     <width>161</width>
     <height>30</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>20</height>
    </size>
   </property>
   <property name="text">
    <string>SAVE</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
